zoo_project.py

```diff
@@ -85,80 +107,350 @@ class Animal:
     def __init__(self, name):
         self.name = name
 
-    def __repr__(self):
+    def __str__(self):
         return "{}".format(self.name)
 
+# Defining preys first as they will be used by the predators
 
-class Lion(Animal):
 
-    """Species class will define the specific characteristics of the animals that are not shared between them
+class Turtle(Animal):
+
+    """Species class will define the specific
+    characteristics of the animals that are not shared between them
     """
+    species = "Turtle"
 
     def __init__(self, name):
-        Animal.__init__(self, name)
-        self.species = "Lion"
-        # If the animal species has known preys, they will be added as the prey property of the animal,
-        #  to be used when the animal is added to the cage.
-        self.preys = ["sheep", "wildebeest", "impala", "zebra", "giraffe", "buffalo", "wild hog", "rhinoceros", 
-                      "hippopotamus"]
+        super().__init__(name)
 
-    def __repr__(self):
+    def __str__(self):
         return "{} the {}".format(self.name, self.species)
```

Comment:
Two comments here.  First, all the animals have the same __init__ and __str__ methods.  You don't need to write these, instead just define them in one place on the Animal class.

Second, why do you need to set the species as a string?  Instead you could just use the class name, i.e. `self.__class__.__name__`.